tests/test_lambdas/test_enums.py

```diff
@@ -0,0 +1,191 @@
+import pytest
+
+from lbz.lambdas import LambdaSources
+
+event_bridge_event = {
+    "version": "0",
```

Comment:
I am pretty sure you don't need 90% of the hardcoded data 😉 

Please, reduce them to minimum to simplify reading the tests.